recipes/pyplis/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyplis" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 06d2e58986efab8bb46a4b5ca27e73790686d83967039732b7746f286e009c66
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
You don't need these just the `yum_requirements.txt` file if you're only using something that depends on OpenGL. In which case, I think you can move this back to being `noarch`?